std/datetime/_constants.ts

```diff
@@ -0,0 +1,6 @@
+// constant times in ms
+export const SECONDS = 1e3; // 1000;
+export const MINUTES = 6e4; // SECONDS * 60;
+export const HOURS = 36e5; // MINUTES * 60;
+export const DAYS = 864e5; // HOURS * 24;
+export const WEEKS = 6048e5; // DAYS * 7;
```

Comment:
it might make more sense for these variables to have singular names, since they're all the number of milliseconds in a single unit.